formidling/src/main/java/no/nav/ung/sak/formidling/vedtak/regler/strategy/EndringBarnDødsfallStrategy.java

```diff
@@ -28,7 +30,7 @@ public final class EndringBarnDødsfallStrategy implements VedtaksbrevInnholdbyg
 
     @Override
     public VedtaksbrevStrategyResultat evaluer(Behandling behandling, LocalDateTimeline<DetaljertResultat> detaljertResultat) {
-        return new VedtaksbrevStrategyResultat(null, "Ingen brev ved dødsfall av barn.", IngenBrevÅrsakType.IKKE_IMPLEMENTERT);
+        return new VedtaksbrevStrategyResultat(null, null, "Ingen brev ved dødsfall av barn.", IngenBrevÅrsakType.IKKE_IMPLEMENTERT);
```

Comment:
Det er litt vanskelig å lese kva som sendes inn som null her. Eg ville kanskje ha definert ein statisk metode for å generere dette, f.eks. VedtaksbrevStrategyResultat.utenBrev()

Reply:
fixed